mmv1/third_party/terraform/.teamcity/components/packages.kt

```diff
@@ -0,0 +1,11 @@
+// this file is copied from mmv1, any changes made here will be overwritten
+
+var packages = mapOf(
+        "acctest" to "AccTest",
+        "services" to "Services",
+        "tpgdclresource" to "TPG DCL Resource",
+        "tpgiamresource" to "TPG IAM Resource",
+        "tpgresource" to "TPG Resource",
+        "transport" to "Transport",
+        "verify" to "Verify"
+)
```

Comment:
`provider` and `fwprovider` are new added packages and need to be added to the list here, I think. They can also be added in a separate PR later.